django_python3_ldap/management/commands/ldap_sync_and_deactivate_users.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from django_python3_ldap import ldap
+from django_python3_ldap.conf import settings
+from django.contrib.auth.models import User
```

Comment:
I tried to run this scrupt but, after read all activate users and put them into the list (line 29) and take the difference (line 31), it shows a error: Manager isn't available...

I solved it by change this line (6) to:
```
from django.contrib.auth import get_user_model
User = get_user_model()
```

Reply:
thanks! ofcourse there needs to be support for a custom User model.

fixed in: https://github.com/busla/django-python3-ldap/commit/4ae7ab3f3765bf0e4b9ea80377964c1674e37de0